syndesis/tests/org.fusesource.ide.syndesis.extension.core.tests/src/test/java/org/fusesource/ide/syndesis/extensions/core/model/SyndesisExtensionUtilInvalidVersionTest.java

```diff
@@ -30,14 +30,19 @@
 		return new String[] {
 				"1",
 				"1-SNAPSHOT",
-				"1.1.SNAPSHOT",
-				"1.1-SNAPSHOT",
-				"1.1.0-SNAPSHOT",
-				"1.1.v20180108",
```

Comment:
why this use case has been removed?

Reply:
We need valid maven versions here. As this value is a valid maven version it has to be removed from the InvalidVersionsTest.